feedback/Dockerfile

```
@@ -0,0 +1,34 @@
+# Copy dependencies first (layer-cached unless requirements change)
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy the shared Pretalx client from parent directory (copied by run.sh)
+COPY pretalx_client.py .
+
+# Copy fonts (copied by run.sh)
+COPY fonts/ static/fonts/
+
+# Copy application code
+COPY app.py db.py auth.py pretalx_cache.py models.py data_loader.py .
```

Hadolint (2.15.1)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)